charts/microgateway/README.md.gotmpl

```diff
@@ -226,431 +236,399 @@ In case that the [Simple DSL configuration](#simple-dsl-configuration) does not
+        MIIEpAIBAAKCAQEAtXkTjHtDtutxyo1R6N4Eh18IzxoagAHPRzsdB5yeadcVr/bV
+        [...]
+        pGKu7aodwB4cD5YnfXTvUcTv5tNU0llRLG1J0bg1n9cCo0nTC9sUZw==
+        -----END RSA PRIVATE KEY-----
+      destinationCACertificate: | 
+        -----BEGIN CERTIFICATE-----
+        MIIDrIXdz54xcilsKUoepQkn9e0bmIUVuiXWcQrr8iqjYC+hINNmiq+4YX4lWq2M
+        [...]
+        K0RRA/rDxZnkbvtTd+hkoMu3Or+pqpOrp2n1pbtzoVl9Hg==
+        -----END CERTIFICATE-----
+  ```
+
+##### Route Passthrough configuration
+With the TLS termination type "Passthrough", HTTPS traffic is sent directly to the Microgateway, without decrypting it on the Route.
+Therefore, no certificates need to be configured on the Route and termination takes place in the Microgateway.
+
+  To setup Passthrough TLS termination, use the following configuraion:
+  ```
+  route:
     enabled: true
-    termination: passthrough
-```
+    hosts:
+      - virtinc.com
+    tls:
+      enabled: true
+      termination: passthrough
+  ```
 
-## Environment variables
-With the Helm chart environment variables can be set.   
-These can then be used in the DSL configuration.   
-Below is an example of how to set environment variables and then use them in the DSL.   
-
-env-variables.yaml
-```
-config:
-  global:
-    env:
-      - name: WAF_CFG_OPERATIONALMODE
-        value: production
-      - name: WAF_CFG_LOGONLY
-        value: false
-```
-
-custom-values.yaml
-```
-config:
-  simple: 
-    mapping:
-      operationalMode: "@@WAF_CFG_OPERATIONALMODE@@"
-      denyRules:
-        logOnly: "@@WAF_CFG_LOGONLY@@"
-```
+## Security
+The following subchapters describes how use and deploy the Microgateway in a secure manner.
 
-## Probes
-In Kubernetes and Openshift probes are used to determine if a pod is ready to process requests or not.  
+### Store sensitive information in secrets
+Airlock Microgateway uses a few sensitive information which should be protected accordingly. The way to do that in Kubernetes or Openshift environments is, to store them in secrets.
+The following subchapters describe which information should be protected and how this can be achieved.
 
-### Liveness Probe
-The liveness Probe is used to find out when to restart a Pod.   
-If a liveness Probe fails, the Pod will restart.   
+#### Secure handling of license and passphrase
+It is possible to use the following parameters of this Helm chart to configure license and passphrase:
+* License: `config.generic.license`
+* Passphrase: `config.generic.passphrase`
 
-The Helm chart has already predefined a liveness probe.   
-If this does not work, it can be adjusted as follows.   
+The Helm chart itself creates a secret and configure the Microgateway to use it. While this is already secure, because it is stored as secrets, these information might end in a Git repo or somewhere too many people have access to.
+This is why it would be much better to create a secret containing license and passphrase with a different process.
```

Comment:
This is why it would be much better -->
This is why it is be much better

Reply:
Done